views/pages/courses/index.ejs

```diff
@@ -1,87 +1,55 @@
 <!--Some arrays of objects for displaying purposes, once there is course data in the database, we will pull from that information instead-->
-<% currentCourses = [{
-  number: "MATH 158",
-  name: "Calculus 3",
-  credit: 4,
-  more: {
-    grade: 'A',
-    category: ['Mathematical Course', 'Mobile Applications Course'],
-    date: 'Current',
-  }
-},{
-  number: "CS 404",
-  name: "Adv Web and Internet",
-  credit: 3,
-  more: {
-    grade: 'B+',
-    category: ['Mobile Applications Course'],
-    date: 'Current',
-  }
-},{
-  number: "MSCS 234",
-  name: "Machine Learning",
-  credit: 3,
-  more: {
-    grade: 'B',
-    category: ['Mathematical Course', 'Mobile Applications Course'],
-    date: 'Current',
-  }
-}]
-completedCourses = [{
-  number: "MATH 157",
-  name: "Calculus 2",
-  credit: 4,
-  more: {
-    grade: 'A-',
-    category: ['Mathematical Course'],
-    date: 'Fall 2020',
-  }
-},{
-  number: "CS 380",
-  name: "Web and Internet",
-  credit: 3,
-  more: {
-    grade: 'A',
-    category: ['Mathematical Course'],
-    date: 'Spring 2018',
-  }
-},{
-  number: "ENGL 102",
-  name: "English 2",
-  credit: 2,
-  more: {
-    grade: 'C+',
-    category: ['Communication Skills'],
-    date: 'Summer 2019',
-  }
-}]
-%>
+<% currentCourses = [{ number: "MATH 158", name: "Calculus 3", credit: 4, more: { grade: 'A',
+category: ['Mathematical Course', 'Mobile Applications Course'], date: 'Current', } },{ number: "CS
+404", name: "Adv Web and Internet", credit: 3, more: { grade: 'B+', category: ['Mobile Applications
+Course'], date: 'Current', } },{ number: "MSCS 234", name: "Machine Learning", credit: 3, more: {
+grade: 'B', category: ['Mathematical Course', 'Mobile Applications Course'], date: 'Current', } }]
+completedCourses = [{ number: "MATH 157", name: "Calculus 2", credit: 4, more: { grade: 'A-',
+category: ['Mathematical Course'], date: 'Fall 2020', } },{ number: "CS 380", name: "Web and
+Internet", credit: 3, more: { grade: 'A', category: ['Mathematical Course'], date: 'Spring 2018', }
+},{ number: "ENGL 102", name: "English 2", credit: 2, more: { grade: 'C+', category: ['Communication
+Skills'], date: 'Summer 2019', } }] %>

```

Comment:
This formatting will break the array

Reply:
I did not chage anything in this file. My OCD just made me save this by habit. All changes from my branch in this file can be deleted

Reply:
I can change it back real quick

Reply:
Yeah I found out the fun way that the prettier formatting breaks the array each time I would format it lol